src/nft/helpers.ts

```diff
@@ -45,7 +45,7 @@ export const nftsFromOperations = (ops: Operation[]): NFT[] => {
       return acc;
     }, {});
 
-  return Object.values(nftsMap);
+  return Object.values(nftsMap).filter((n) => n.amount.gt(0));
```

Comment:
No risk of empty values in `nftsMap`?
If so should do `Object.values(nftsMap).filter(Boolean).filter((n) => n.amount.gt(0))` otherwise it's fine.

Reply:
mmh to me no risk. `nft` is never falsy. in that `acc[nftKey] = nft;`